tests/drivers/gpio/egpio_basic_api/src/test_egpio.h

```diff
@@ -9,7 +9,6 @@
 
 #include <zephyr/kernel.h>
 #include <zephyr/drivers/gpio.h>
-#include <zephyr/sys/util.h>
```

Comment:
@jaz1-nordic Is it safe to remove this line? Locally, I see failure in stress test for the icbmsg backed but I doubt it is related. Or, is it?

Reply:
I think it can be removed. It's a leftover from testing in zephyr. What errors do you have for icbmsg?

Reply:
In basic_egpio_test I add one test case - a "stress" test.
Yesterday, I got test FAILED because when loop ends, rc was huge negative number like -2222546.
Thus, I added in the loop TC_PRINT() when rc is non zero. And now, I get timeout.
https://jenkins-ncs.nordicsemi.no/job/latest/job/sub/job/test-low-level/job/main/3236/testReport/junit/nrf54l15dk_nrf54l15_cpuapp_egpio_basic_api_drivers.egpio/loopback_icbmsg/Build_and_test___Moonlight_App_core___Moonlight_App_core___Build_and_run_Zephyr_tests___egpio_basic_api_drivers_egpio_loopback_icbmsg/

Reply:
When ICBMSG is used, at 16th iteration of the FOR loop,
gpio_pin_set_raw() returns -5 (-EIO)
```
- stress_gpio_pin_set_raw
16: rc = -5

    Assertion failed at WEST_TOPDIR/nrf/tests/drivers/gpio/egpio_basic_api/src/test_egpio_port.c:481: stress_gpio_pin_set_raw: (rc not equal to 0)
at least one set operation failed, rc = -5)
```
As if ICBMsg doesn't block/pause code execution when there is no more space in the IPC buffers.